app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/ContactSA.scala.html

```diff
@@ -39,57 +39,59 @@
 }
 
 @content = {
-<p class="govuk-body">@messages("reportSuspiciousId.detailsComponent.text1")</p>
-<p class="govuk-body">@messages("reportSuspiciousId.detailsComponent.text2")
-<a href="https://www.relayuk.bt.com/">@messages("reportSuspiciousId.detailsComponent.link")</a>
-@messages("reportSuspiciousId.detailsComponent.text3")</p>
-<p class="govuk-body">@messages("reportSuspiciousId.detailsComponent.text4")</p>
+<p class="govuk-body">@messages("contactSA.detailsComponent.text1")</p>
+<p class="govuk-body">@messages("contactSA.detailsComponent.text2")
+<a class = "govuk-link" target="_blank" href="https://www.relayuk.bt.com/">@messages("contactSA.detailsComponent.link")</a>
+@messages("contactSA.detailsComponent.text3")</p>
+<p class="govuk-body">@messages("contactSA.detailsComponent.text4")</p>
 }
 
 
 @layout(
-pageTitle = Some(messages("reportSuspiciousId.title")), Some("?")) {
+pageTitle = Some(messages("contactSA.title")), Some("?")) {
+
+<h1 class="govuk-heading-xl">@{messages(s"contactSA.heading")}</h1>
+
+<p class="govuk-body">
+    @messages("contactSA.text1")
+    @messages("contactSA.text2")
+</p>
 
-<h1 class="govuk-heading-xl">@{messages(s"reportSuspiciousId.heading")}</h1>
 <dl class="govuk-summary-list">
 
-    @createRowWith(messages("reportSuspiciousId.userId"), ptAccountDetails.userId)
-    @createRowWith(messages("reportSuspiciousId.email"), ptAccountDetails.email.get)
-    @createRowWith(messages("reportSuspiciousId.lastSignedIn"), ptAccountDetails.lastLoginDate)
```

Comment:
I assume this was an unintentional deletion? We still need the last signed in info.

Reply:
It's intentional. as this is cover by logic created by Christine and in this view has been used. 
 @for(mfaDetail <- ptAccountDetails.mfaDetails) {
    @createRowWith(mfaDetail.factorName, mfaDetail.factorValue)